lib/assembly/convert-circuit-json-to-assembly-svg.ts

```diff
@@ -166,6 +166,7 @@ function createSvgObjects(
             transform,
             { x: firstPort.x, y: firstPort.y },
             sourceComponent.name,
+            sourceComponent.ftype,
```

Comment:
FWIW instead of stacking more position parameters onto this function, it should be converted to a two parameter function https://github.com/tscircuit/handbook/blob/main/guides/code.md#1-two-parameter-rule